tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py

```diff
@@ -144,62 +140,3 @@ def test_ingest_consumer_reads_from_topic_and_calls_celery_task(
     assert transaction_message.data["event_id"] == transaction_event_id
     assert transaction_message.data["spans"] == []
     assert transaction_message.data["contexts"]["trace"]
-
-
-@django_db_all(transaction=True)
-def test_ingest_topic_can_be_overridden(
```

Comment:
test is obsolete since the CLI and topic overrides are not an ingest-consumer concern anymore